docs/clip_media_sim.md

```diff
@@ -0,0 +1,56 @@
+# CLIPMediaSim
+
+## Video similarity
+
+```python
+from vision_agent_tools.tools.clip_media_sim import CLIPMediaSim
+from decord import VideoReader
+from decord import cpu
+from PIL import Image
+
+# Path to your target image
+image_path = "path/to/your/image.jpg"
+
+# Path to your video
+video_path = "path/to/your/video.mp4"
+
+# Load the image
+target_image = Image.open(image_path)
+
+# Load the video
+vr = VideoReader('va-generation.mp4', ctx=cpu(0))
```

Comment:
@camiloaz please change this to `video_path`

Reply:
Thanks! Fixed